scalafix-tests/unit/src/test/scala/scalafix/tests/cli/ScalafixImplSuite.scala

```diff
@@ -97,6 +99,47 @@ class ScalafixImplSuite extends AnyFunSuite with DiffAssertions {
     assert(e.isPresent)
     assert(e.get().getMessage.contains("-Ywarn-unused"))
   }
+  test("configure scalafix path") {
```

Comment:
i don't understand this description - isn't the test more about
```suggestion
  test("rulesThatWillRun") {
```